packages/wrangler/src/api/startDevWorker/types.ts

```diff
@@ -51,7 +60,7 @@ export interface StartDevWorkerOptions {
 	 * This is the `main` property of a wrangler.toml.
 	 * You can specify a file path or provide the contents directly.
 	 */
-	entrypoint: FilePath;
+	entrypoint?: FilePath;
```

Comment:
What does it mean for a worker to have no entry-point?

Reply:
As discussed offline, it means you will have to have a `main` field in your config file. There is existing runtime code to check one of the possible ways to provide an entrypoint is actually provided and throws if not. The existing code was a bit too hairy to create a static type union but will be possible in the future as more deadcode paths are removed.